optuna/storages/base.py

```diff
@@ -171,6 +171,11 @@ def get_all_trials(self, study_id):
 
         raise NotImplementedError
 
+    def unsafe_get_all_trials(self, study_id):
```

Comment:
How about adding `deepcopy` option to `BaseStorage.get_all_trials` method instead of adding new (similiar) method?